packages/nimble-components/src/rich-text/viewer/styles.ts

```diff
@@ -25,6 +25,7 @@ export const styles = css`
         box-sizing: border-box;
         position: relative;
         color: inherit;
+        word-break: break-word;
```

Comment:
Quickly skimmed through MDN docs for [`word-break`](https://developer.mozilla.org/en-US/docs/Web/CSS/word-break) and [`overflow-wrap`](https://developer.mozilla.org/en-US/docs/Web/CSS/overflow-wrap). 

Looks like `word-break:break-word` is deprecated. `overflow-wrap: break-word` seems to be a good alternative

![image](https://github.com/ni/nimble/assets/94439533/01bf6d74-3cc6-4978-82b9-bc5948a583bc)

```suggestion
        overflow-wrap: break-word;
```

Reply:
`overflow-wrap: break-word;` seems to have a different behavior where it brings a scroll bar for a single long word instead of breaking it and breaks only if new words are added.
![image](https://github.com/ni/nimble/assets/123377167/e64d4174-f718-49c2-8de6-4a58419c2c8a)

But `overflow-wrap: anywhere;` has the same behaviour as ` word-break: break-word;`